gatsby-config.js

```diff
@@ -5,6 +5,14 @@ module.exports = {
     author: `@gatsbyjs`,
   },
   plugins: [
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        trackingId: "UA-134138489-1", 
```

Comment:
I'm pretty sure Mr. Bascom wanted this to be passed in through an environment variable. Double-check with him to make sure this is ok.

Reply:
@oconnelc @agrimreaper49 Yeah, I wanted to avoid adding the id to the public repo, but there it is. I would still like to add this as an environment variable.